Zend/zend_compile.c

```diff
@@ -6788,10 +6788,10 @@ static zend_type zend_compile_typename_ex(
 }
 /* }}} */
 
-static zend_type zend_compile_typename(zend_ast *ast, bool force_allow_null)
+static zend_type zend_compile_typename(zend_ast *ast)
 {
 	bool forced_allow_null;
-	return zend_compile_typename_ex(ast, force_allow_null, &forced_allow_null);
+	return zend_compile_typename_ex(ast, false, &forced_allow_null);
 }
```

Comment:
It might make sense to port this change to master already, regardless of the RFC result, as this is a nice simplification

Reply:
Ported to master and rebased on top of it!